### PR TITLE
Refactor classification module

### PR DESCRIPTION
**Pull request by roshankern**
This PR is ready for review!

In this PR, the classification is refactored to include classifications for models with CP features and single-cell models.

### CODE REVIEW

File: 4.classify-single-cell-phenotypes/classification-utils.py [hunk 1]
```diff
@@ -8,70 +8,29 @@
...
-    Parameters
-    ----------
-    phenotypic_profiling_model : LogisticRegression
-        model to use for classification
-    normalized_plates_path : pathlib.Path
-        path to normalized plates
-    output_dir : pathlib.Path
-        where to save features classifications
-    """
+def get_probas_dataframe(plate_features, model, feature_type):
```
Comment (jenna-tomkinson) [inline 4.classify-single-cell-phenotypes/classification-utils.py]: Recommend adding a doc string here.

Comment (jenna-tomkinson) [inline 4.classify-single-cell-phenotypes/classification-utils.py]: Recommend adding type hints here.

**Review by jenna-tomkinson — APPROVED** [review]
Great PR! Just added two comments to suggest adding more documentation to the function in the utils file.

File: 4.classify-single-cell-phenotypes/classification-utils.py (hunk 1 above)
Comment (roshankern) [reply 4.classify-single-cell-phenotypes/classification-utils.py]: Done in https://github.com/WayScience/cell-health-data/pull/16/commits/87bf3b49e57b36410caba8612063de42d8f265e2

Comment (roshankern) [reply 4.classify-single-cell-phenotypes/classification-utils.py]: Done in https://github.com/WayScience/cell-health-data/pull/16/commits/87bf3b49e57b36410caba8612063de42d8f265e2

**Pull request merged**